tests/testColor.py

```diff
@@ -308,11 +275,6 @@ def defineFilterProperty(self, name, lambdaEff, force=False):
     def testListFilters(self):
         self.assertEqual(afwImage.Filter_getNames(), self.filters)
 
-    def testCtor(self):
```

Comment:
It looks like this functionality is already in several other tests, including the an exact replica in `testFilterEquality`, so I think it's ok to leave this deleted.